tools/link.py

```diff
@@ -621,6 +621,24 @@ def set_max_memory():
     exit_with_error('MAXIMUM_MEMORY cannot be less than INITIAL_MEMORY')
 
 
+def check_browser_versions():
+  settings_map = {
```

Comment:
```suggestion
  oldest_map = {
```
This is really a map of VM to the oldest supported version, so "oldest" in the name seems clearest?